mobile/src/screens/ProfileScreen.tsx

```diff
@@ -0,0 +1,201 @@
+ */
+const SettingsTab: React.FC = () => {
+
+  const i18nCtx = useContext(I18nContext);
+  const { i18n, setLocale, t, locale } = i18nCtx;
+  const languages = Object.keys(i18n.translations) as Language[];
+
+  return (
+    <View>
+		
+      <View style={styles.login}>
+        <LargeButton text="Sign In" onPress={() => null} underline={true}/>
+        <LargeButton text="Sign Up" onPress={() => null} underline={true}/>
+      </View>
+
+      <View style={styles.langSelector}>
+
+        <View style={styles.languageText}>
+          <Text style={TextStyles.heading3}>{t("language")}</Text>
+        </View>
+				
+        {languages.map((lang: Language) => (
+          <View key={`lang${lang}`} style={styles.langElem}>
+						
+            <Text style={TextStyles.body1}>{Languages[lang]}</Text>
+
+            <Pressable
+              onPress={() => setLocale(lang)}
+              style={styles.box}
+            >
+              { lang === locale && <Image style={styles.boxChecked} source={require('../../assets/images/check-square-solid.png')}/>}
+            </Pressable>
+
+          </View>
+        ))}
+
+      </View>
+
+      <View style={{height:300}}/>
+
+    </View>
+	
+	
+  );
+
+};
+
+const MoreInfoTab: React.FC = () => {
+  const i18nCtx = useContext(I18nContext);
+  const { t } = i18nCtx;
+
+  const [value, onChangeText] = React.useState('');
+
+  return (
+    <View style={{ alignSelf: 'center', width: 298, alignItems: 'center'}}>
+      <Text style={[TextStyles.heading3, {marginBottom: 20}]}>{t("socialMedia")}</Text>
+      <View style={styles.socialRow}>
+        <TouchableOpacity onPress={async () => await Linking.openURL("https://twitter.com/WordsAliveSD")}>
+          <Image style={styles.socialPic} source={require('../../assets/images/twitter.png')}/>
+        </TouchableOpacity>
+        <TouchableOpacity onPress={async () => await Linking.openURL("https://instagram.com/wordsalivesd")}>
+          <Image style={[styles.socialPic, styles.notLeftPic]} source={require('../../assets/images/instagram.png')}/>
+        </TouchableOpacity>
+        <TouchableOpacity onPress={async () => await Linking.openURL("https://www.facebook.com/WordsAliveSD")}>
+          <Image style={[styles.socialPic, styles.notLeftPic]} source={require('../../assets/images/facebook.png')}/>
+        </TouchableOpacity>
+        <TouchableOpacity onPress={async () => await Linking.openURL("https://www.tiktok.com/@wordsalivesd")}>
+          <Image style={[styles.socialPic, styles.notLeftPic]} source={require('../../assets/images/tiktok.png')}/>
+        </TouchableOpacity>
+      </View>
+      <LargeButton text={t("donate")} onPress={async () => await Linking.openURL("https://www.wordsalive.org/donate")}/>
+      <LargeButton text={t("volunteer")} onPress={async () => await Linking.openURL("https://www.wordsalive.org/becomeavolunteer")}/>
+      <Text style={[TextStyles.heading1, {textAlign: 'center', marginTop: 20}]}>
```

Comment:
Let's drop this down to h3 now, since the text is so much longer and that's what the mockup shows